src/index.html

```diff
@@ -36,8 +36,10 @@
         // Gets the URL for the API
         const GET_API_BASE_URL = () => {
           let URL = window.location.origin;
-          if (!window.location.origin.includes('localhost')) {
-            // include project ID in prod
+          const isLocalhost = window.location.origin.includes('localhost');
+          const isRun = window.location.origin.includes('run.app');
+          if (!isLocalhost && !isRun) {
+            // include project ID in URL if GCF (e.g. not localhost or Run)
             URL += '/tasks-pizza';
```

Comment:
Out of curiosity, why does GCF uniquely need the project ID? If `tasks-pizza` is a project ID is it safe to hard-code?

Reply:
See go/path-manipulation-in-google-function-frameworks for details.